index.js

```diff
@@ -213,25 +212,36 @@ function isFastifyMultipartRegistered (fastify) {
   return fastify.hasContentTypeParser('multipart') && fastify.hasRequestDecorator('multipart')
 }
 
-function createRequestRetry (requestImpl, reply, retriesCount, retryOnError) {
+function createRequestRetry (requestImpl, reply, retriesCount, retryOnError, retryOnCode = 503) {
```

Comment:
```suggestion
function createRequestRetry (requestImpl, reply, retriesCount, retryOnError, retryOnCode) {
```

Reply:
After some thoughts I will remove that, the issue is specifically for 503, and I would not make it configurable by the user since we may hit an external server.